src/check.nim

```diff
@@ -10,6 +10,10 @@ proc check*(conf: Conf) =
     case exercise.status
     of exOutOfSync:
       logNormal(&"[warn] {exercise.slug}: missing {exercise.tests.missing.len} test cases")
+      let missingCases = exercise.testCases
+        .filterIt(it.uuid in exercise.tests.missing)
+      for testCase in missingCases:
+        logNormal(&"       - {testCase.description} ({testCase.uuid})")
```

Comment:
```suggestion
      for testCase in exercise.testCases:
        if testCase.uuid in exercise.tests.missing:
          logNormal(&"       - {testCase.description} ({testCase.uuid})")
```

I personally prefer this, though. It allocates less, and I find it more readable.

I'm sympathetic to functional programming style, but `sequtils` procs require an import and currently generate inefficient code. That might be improved in the future, though (see a Nimble package called [zero-functional](https://github.com/zero-functional/zero-functional#initial-example)).

And I actually have a branch where I remove the repeated loop due to the `anyIt` at the bottom of this file :)

Reply:
@glennj Are you okay with the above change? If so, could you apply it and then I'll merge.